src/Compilers/Core/Portable/Symbols/ISourceAssemblySymbol.cs

```diff
@@ -16,7 +16,6 @@ namespace Microsoft.CodeAnalysis
     /// This interface is reserved for implementation by its associated APIs. We reserve the right to
     /// change it in the future.
     /// </remarks>
-    [InternalImplementationOnly]
```

Comment:
It was the only location besides ISymbol that used the attribute. Feels not
right, why have it only here?